### DISCUSSION TITLE
Some mode adjustments should be permanent

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#29580] [Ideas]
I can't get a reason to reset mod adjustments to initial values - it's difficult to configure them again for every single mod. Initial values are almost always not the best ones. Presets can't help to solve this problem as a lot of mod combinations can exist.

The most useful case of permanent adjustment can be Pitch Adjust switch on DT mode - it can solve dilema of two players who hate and love nightcore pitch. There will be no problems if one player will play with this adjustment enabled by default and second without as song duration will be the same. The same story with spectator - song tone will depend on Pitch switch state.

**Comment by Tenexxt** [comment]
You can use mod presets for that

**Reply by okolenmi** [reply]
You answered to the first 22 words. Can you read more before you clutter this thread with your nonesense? There can be thousands of mod combinations.

**Reply by Tenexxt** [reply]
Enabling DT pitch adjustments require you literally only one click, since now the customisation dropdown opens automatically.

As for watching a replay with DT/HT, adding an option to turn on the pitch adjust for the replay would be nice, and the option to turn off certain mods for replays (like HD) actually has a pull request open, but it's currently inactive.

While you can have thousands of mod combinations, let's be honest, most of the time you will use a certain combination, like DTHD, and this can be easily solved by presets, and as I already said - turning on certain toggles for mods takes literally one click and less than a second if you're fast.

Tho I'll admit that some mods should have different default settings ( Sudden Death having restart after fail )

**Reply by okolenmi** [reply]
> - option to turn off certain mods for replays (like HD) actually has a pull request

Are you a bot? Where exactly I have mentioned this? Even ChatGPT 3 have more capabilities than you (you are near GPT2). 

> most of the time you will use a certain combination, like DTHD

It's not old osu, you know... I'm using speed change on DT almost always, approach circle animations and other mods for fun/practice. What a reason to write this topic if the only thing I'm using is HDDT?

Why are you talking to me again without reading whole topic properly? You are too lazy to read some text, but instead have a lot of energy to ruin a good concept. The topic is about all adjustments that have no effect on difficulty. It's not about pitch only (it was an example). 

If it's difficult for you to understand - **the main point is mods/adjustments separation into difficulty and non-difficulty parts.** It can create a way to solve problem of players, who want to play with pitch and without in the same lobby, along with other benefits.
**Separation of mod adjustments** should have been don before they was implemented as it's silly to put them all together. Mods/Mod adjustments should be separated into difficulty and "cosmetic" parts. Cosmetic part should be enabled always if possible.

**Reply by Tenexxt** [reply]
You literally Mentioned spectating people in multiplier, and since it applies for multi at also must apply for replays.

The only use of the "good concept" is DT and HT pitch adjust because the most of the rest affect the difficulty and make the mod unranked and could cause problems in multiplier, since you'd be essentialy watching a different game.

Some Cosmetic changes also technically affect Gameplay, like Approach Different where you can choose how the circles approach, and different modes will be easier or harder to read for some people.

As I already said multiple times: Enabling a mod and it's adjustment will take you max 10 seconds when you already know what you want to play with, if it takes more, you most likely didn't know what you want to play with, which is obviously a you problem.

Also the fact that you accuse me of boting because I don't like your concept and my English is bad doesn't make you look good btw.
